pipeline/data/download-mono.py

```diff
@@ -133,20 +133,37 @@ def main(args_list: Optional[list[str]] = None) -> None:
         ):
             outfile.write(line)
 
+    handle_chinese(args, file_destination)
+
+
+def handle_chinese(args, file_destination):
     # TODO: convert everything to Chinese simplified for now
     # TODO: https://github.com/mozilla/firefox-translations-training/issues/896
     if args.language == "zh":
-        logger.info("Converting the output file to Chinese Simplified")
-        chinese_converter = ChineseConverter()
-        converted_path = file_destination.with_suffix(".converted.zst")
-        stats = chinese_converter.convert_file(
-            file_destination, converted_path, ChineseType.simplified
-        )
-        shutil.move(converted_path, file_destination)
-        print(
-            f"Converted {stats.script_conversion.converted} lines from {stats.script_conversion.visited} to Chinese Simplified"
-        )
-        stats.save_json()
+        if os.environ["SRC"] == "zh":
+            logger.info("Converting the output file to Chinese Simplified")
+            chinese_converter = ChineseConverter()
+            converted_path = file_destination.with_suffix(".converted.zst")
+            stats = chinese_converter.convert_file(
+                file_destination, converted_path, ChineseType.simplified
+            )
+            shutil.move(converted_path, file_destination)
```

Comment:
I guess this move overwrites the file_destination file, so we don't retain extra copies of the datasets. It might be more explicit to remove the old one and then move so it's clearer. I had to look up this API.